WordPressAuthenticator/Navigation/NavigateToEnterAccount.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+public struct NavigateToEnterAccount: NavigationCommand {
+    public init() {}
+    public func execute(from: UIViewController?) {
```

Comment:
WPAuthenticator inserts the actual view controller that triggers this command, so WPAuthenticator knows that this requires manipulating a navigation controller